test/unit.test.js

```diff
@@ -34,18 +34,18 @@ describe('matchBrowserLocale', () => {
     const { matchBrowserLocale } = await import('../src/templates/utils-common')
 
     // Both locales match first browser locale - full locale should win.
-    appLocales = ['en', 'en-US', 'en-us']
-    browserLocales = ['en-US', 'en']
+    string[] appLocales = ['en', 'en-US', 'en-us']
+    string[] browserLocales = ['en-US', 'en']
```

Comment:
Sorry, but the changes you are making make no sense. You don't seem to have an understanding of javascript. Maybe you want me to just take over that fix?

Reply:
Haha, indeed I was a programmer decades ago but I am doing it to fight against COVID-19!

Reply:
If you take over the change, I much appreciate it. Thank you very much!

Reply:
@mcdmaster He might want you to leave the tests that have already been as is, and to create just new tests about case-insentive comparison

Reply:
@rchl Can you please be handed over to create testing?

@MaySoMusician I think that this implementation needs deep dive into existing behavior,, which should not be spoiled. I am convinced that @rchl is the most aware of what `nuxt-i18n` should be with our request implemented. Hence, I wanted to hand this over to him. Thank you anyway